job-service/src/main/java/com/hpe/caf/services/job/queue/QueueServices.java

```diff
@@ -88,14 +90,6 @@ public void sendMessage(
             } else {
                 throw new RuntimeException("Unknown taskDataEncoding");
             }
-        } else if (taskDataObj instanceof Map<?, ?>) {
```

Comment:
I don't get why this is being dropped or how that squares with the code above.

Reply:
Restored